flink-clients/src/test/java/org/apache/flink/client/CliFrontendRunTest.java

```diff
@@ -100,15 +100,22 @@ public void testRun() {
 			}
 
 			// test jar arguments
+            {
+                String[] parameters =
+                        {"-m", "localhost:6123", getTestJarPath(), "-arg1", "value1", "justavalue", "--arg2", "value2"};
+                RunOptions options = CliFrontendParser.parseRunCommand(parameters);
+                assertEquals("-arg1", options.getProgramArgs()[0]);
+                assertEquals("value1", options.getProgramArgs()[1]);
+                assertEquals("justavalue", options.getProgramArgs()[2]);
+                assertEquals("--arg2", options.getProgramArgs()[3]);
+                assertEquals("value2", options.getProgramArgs()[4]);
```

Comment:
Tabs have been converted to spaces here. That's why the diff looks odd. Please use tabs for indention.